### ISSUE TITLE
MRU should only apply to CTRL+TAB, not when closing tabs.

### ISSUE THREAD

**Issue by maxime-tanuki** [notepad-plus-plus/notepad-plus-plus#9690]
Under Preferences > MISC., there is an option to 'Enable MRU behavior' when using 'CTRL+TAB'. It works fine, but it also applies when closing the tabs. In older versions of notepad++ this was not the case.

MRU with CTRL+TAB is convenient when comparing or copying content between two files. 

However I sometimes want to clean a session of work and close a bunch of tabs. In that case, it's very confusing to focus tabs in recently used order. I much preferred the old behavior which is to focus the next time on the right when a tab is closed.

In other words, there should be a separate option to control MRU when closing tabs, or only apply MRU when using  'CTRL+TAB'.

**Comment by sasumner** [comment]
Existing behavior was the decision made on how it should function.

**Comment by mjmaax** [comment]
![image](https://user-images.githubusercontent.com/6823305/112335544-ad7afe00-8cff-11eb-81ed-32950873c5c7.png)
groupbox states it's for 'CTRL+TAB'. Talked with several developers who also found it confusing. Suggest to at least edit the title if the intended behavior is different.
